eng/Version.Details.xml

```diff
@@ -174,9 +174,9 @@
       <Sha>5cda86493ac07dce11dcb04323d2b57eecff00b7</Sha>
       <SourceBuild RepoName="emsdk" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-8.0.100" Version="8.0.0-preview.3.24060.4">
+    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-8.0.100" Version="8.0.0-preview.4.24079.1">
```

Comment:
Isn't this dependency superseded by `Microsoft.NET.Sdk.Aspire.Manifest-9.0.100-preview.1`?

Reply:
Yes, it should be, not sure why I didn't also remove this. I'll push up a change that fixes this.